### PR TITLE
Add a couple of unit tests

### PR DESCRIPTION
**Pull request by rudlorenz**

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -233,4 +233,51 @@ mod bitmap_tests {
         bitmap.write(cursor).unwrap();
         assert_eq!(buffer.as_slice(), &0b1001_0010_0100_1001u16.to_ne_bytes());
     }
+
+    #[test]
```
Comment (npatsakula) [inline src/lib.rs:237]:
```suggestion
    #[test]
    fn value_write() {
        let mut bitmap = DynBitmap::contained(32);
        let mut cursor: std::io::Cursor<Vec<u8>> = Default::default();

        for i in 0 .. bitmap.bits_capacity()-1 {
            bitmap.set(i).unwrap();
            bitmap.write(&mut cursor).unwrap();

            assert_eq!(
                ((1u32 << i as u32 + 1) - 1).to_le_bytes().as_ref(),
                cursor.get_ref().as_slice(),
            );

            cursor.set_position(0);
        }
    }
```

File: src/lib.rs [hunk 2]
```diff
@@ -114,7 +114,7 @@ impl DynBitmap {
     /// ```
     pub fn set(&mut self, bit_index: usize) -> Result<(), Error> {
         let byte = self.get_byte_mut(bit_index)?;
-        let position_in_byte = Self::position_in_byte(bit_index) as u8;
+        let position_in_byte = Self::position_in_byte(bit_index);
```
Comment (npatsakula) [inline src/lib.rs:117]: :sunglasses:

**Review by npatsakula — CHANGES_REQUESTED** [review]
:kissing_heart:

File: src/lib.rs (hunk 2 above)
Comment (rudlorenz) [reply src/lib.rs:117]: `position_in_byte` already returns a `u8`

File: src/lib.rs (hunk 1 above)
Comment (rudlorenz) [reply src/lib.rs:237]: \+

**Review by npatsakula — APPROVED** [review]

**Pull request merged**